src/main/com/homework/week9/invoice/service/InvoiceType.java

```diff
@@ -0,0 +1,25 @@
+package com.homework.week9.invoice.service;
+
+public enum InvoiceType {
+    SAD("SAD"),
+    MAN("MAN");
+
+
+    public String getName() {
+        return this.name;
+    }
+
+    private String name;
+
+    InvoiceType(String sad) {
+        this.name = name();
+    }
+
+    public static InvoiceType getEnumByString(String code) {
```

Comment:
the enum has valueOf(String arg) method that does the same thing you wrote. please use it instead of getEnumByString